Hobbies.java

```diff
@@ -0,0 +1,32 @@
+package org.example.homework;
+
+public class Hobbies {
```

Comment:
```suggestion
public class Hobby {
```
class name should be a singular noun